library/std/src/time.rs

```diff
@@ -461,13 +469,22 @@ impl fmt::Debug for Instant {
 impl SystemTime {
     /// An anchor in time which can be used to create new `SystemTime` instances or
     /// learn about where in time a `SystemTime` lies.
+    //
+    // NOTE! this documentation is duplicated, here and in std::time::UNIX_EPOCH.
+    // The two copies are not quite identical, because of the difference in naming.
     ///
     /// This constant is defined to be "1970-01-01 00:00:00 UTC" on all systems with
     /// respect to the system clock. Using `duration_since` on an existing
     /// `SystemTime` instance can tell how far away from this point in time a
     /// measurement lies, and using `UNIX_EPOCH + duration` can be used to create a
     /// `SystemTime` instance to represent another fixed point in time.
     ///
+    /// `duration_since(UNIX_EPOCH).unwrap().as_secs()`
+    /// returns a POSIX `time_t` (as a `u64`):
+    /// the number of non-leap seconds since the start of 1970 UTC.
+    /// This is the same time representation as used in many Internet protocols,
+    /// for example: JWT, CBOR, TOTP, certificate transparency and DNS TSIG/DNSSEC.
```

Comment:
I think enumerating these probably is over the line for too much documentation, since it might lead people to file PRs adding other protocols. I also would prefer saying "This is $description, same as $POSIX_equivalent" rather than the other way around.

So my suggesion would be for this (and the other) comment to be more like:

> `duration_since(UNIX_EPOCH).unwrap().as_secs()` returns the number of non-leap seconds since the start of 1970 UTC, as a u64. This is the same time representation as used in POSIX `time_t`, as well as a wide variety of internet protocols.

(Note: I'm not tied to precise wording except for the bits I mentioned above). Thoughts?

Reply:
I'm OK with these suggestions and have adopted them.  Thanks.